.cookiecutter.json

```diff
@@ -0,0 +1,37 @@
+{
+    "cookiecutter": {
+        "codeowner_github_usernames": "@whitej6 @itdependsnetworks @jdrew82",
+        "full_name": "Network to Code, LLC",
+        "email": "opensource@networktocode.com",
+        "github_org": "nautobot",
+        "plugin_name": "nautobot_firewall_models",
+        "verbose_name": "Nautobot Firewall Models",
+        "plugin_slug": "nautobot-firewall-models",
+        "project_slug": "nautobot-plugin-firewall-models",
+        "repo_url": "https://github.com/nautobot/nautobot-plugin-firewall-models",
+        "base_url": "firewall-models",
+        "min_nautobot_version": "2.0.0",
+        "max_nautobot_version": "2.9999",
+        "nautobot_version": "latest",
+        "camel_name": "NautobotFirewallModels",
+        "project_short_description": "Nautobot App to model firewall and security objects. Allows users to model policies in a vendor-neutral manner and use that data to drive network security automation",
+        "version": "2.0.1",
```

Comment:
Does version come from pyproject.toml or is this a static definition?

Reply:
I vote this needs to be removed.

Reply:
Only seems to be used in these places:

```
{{ cookiecutter.project_slug }}/pyproject.toml:version = "{{cookiecutter.version}}"
{{ cookiecutter.project_slug }}/.github/ISSUE_TEMPLATE/bug_report.md:* {{cookiecutter.plugin_slug}} version:  <!-- Example: {{cookiecutter.version}} -->
{{ cookiecutter.project_slug }}/.github/ISSUE_TEMPLATE/feature_request.md:* {{cookiecutter.plugin_slug}} version:  <!-- Example: {{cookiecutter.version}} -->
```

I guess the version can be hardcoded to 0.1.0 in pyproject for the initial bake then ignored as it will be managed by maintainers.